bitnami/joomla/Chart.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
   tags:
   - bitnami-common
   version: 2.x.x
-description: Joomla! is an award winning open source CMS platform for building websites and applications. It includes page caching, page compression and Let's Encrypt auto-configuration support.
+description: DEPRECATED Joomla! is an award winning open source CMS platform for building websites and applications. It includes page caching, page compression and Let's Encrypt auto-configuration support.
```

Comment:
In other similar PRs I can see:

```
deprecated: true
```

Could you please check it?

Reply:
Good catch! Thanks